### DISCUSSION TITLE
uuid field auto generate issue

### DISCUSSION THREAD

**Discussion by hypong** [prisma/prisma#17094] [Q&A] [answered]
Hi all, I have a problem creating a new record with the UUID field.

this is my error message
```
const createUserData: Promise<User> = users.create(\n  Error in domain logic: Conversion from `Uuid(08b93743-4731-4b2d-a899-5256235a3ffb)` to `Int` failed."
```

this is my table schema 
```
model User {
  id       String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  email    String @unique
  password String
}
```

I have tried another approach
```
id       String @id @default(uuid()) @db.Uuid
```
And I got this error
```
const createUserData: Promise<User> = users.create(\n  Null constraint violation on the fields: (`id`)"
```

this is my dependency 
```
"prisma": "^4.8.0",
"@prisma/client": "^4.8.0",
```

Any suggestions are greatly appreciated
Thanks.

**Comment by hypong** [comment, marked as answer]
I think it's related to my docker-compose setup. Everything its ok now after I rebuild the docker.
Thank all

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
